Documentation/ceph-monitoring.md

```diff
@@ -81,20 +81,22 @@ You can find Prometheus Consoles here: https://github.com/ceph/cephmetrics/tree/
 A guide to how you can write your own Prometheus consoles can be found on the official Prometheus site here: https://prometheus.io/docs/visualization/consoles/.
 
 ## Prometheus Alerts
-To enable prometheus alerts during cluster deployment, make following changes to `cluster.yaml`
+### To enable prometheus alerts
```

Comment:
This L3 header is only needed if there are separate steps for deploying with alerting and enabling alerting after deployment. As it is, there is no distinction, and no L3 subheader is needed.